cradle-admin-tool-http/src/main/java/com/exactpro/th2/cradle/adm/http/Configuration.java

```diff
@@ -35,8 +37,11 @@ public class Configuration {
 	@JsonProperty("page-recheck-interval")
 	private int pageRecheckInterval = DEFAULT_PAGE_RECHECK_INTERVAL_SEC;
 
+	@JsonProperty("auto-book")
+	private boolean autoBook = false;
```

Comment:
I am not sure if we need to disable it by default. It means every time people are going to face a problem with books that do not exist when they start the cradle-admin with auto-page configuration.

Reply:
Ok